### DISCUSSION TITLE
Implement BOINC Central

### DISCUSSION THREAD

**Discussion by davidpanderson** [BOINC/boinc#4965] [General]
Create a BOINC project whose set of applications is those in the app library.

Anyone can create an account and submit jobs to these apps.
- through a web interface (need to develop, per app)
- through a remote cmdline interface (need to develop, generic)
This is level 0 of the "BOINC test drive experience".

Keep track of work done per submitter; use existing scheme to limit/allocate.

Get hosts to attach to this project:
- Volunteers can attach directly
- Science United users can check a box.  SU will attach the users to BOINC Central, with the "restricted" flag.
- try to get institutions to run it as backfill on clusters.

Maintain info about the resource usage of each app and/or app version (RAM, output file size, processor usage).  These may depend on cmdline params and input file lengths or contents.  Maybe do some kind of linear best fit.  This info would be exported via the app library.

**Comment by AenBleidd** [comment]
This doesn't really related to the BOINC application or general BOINC Project server, so I'm moving this to Discussions

**Comment by AenBleidd** [comment]
The Project has been started: https://boinc.berkeley.edu/central/
We're working on engaging scientists to use it.
